### DISCUSSION TITLE
New Package Version Release Cycles

### DISCUSSION THREAD

**Discussion by ConstantinEilebrecht** [mburumaxwell/paklo#1367] [Q&A] [answered]
@rhyskoedijk 
Hi there,
I'm wondering when do new package releases occour normally? Is this a manual process or automated?

Ref my bug #1356 and your fixes from #1362
Can i test something without getting the newest Azure Devops Extensions version?

**Comment by rhyskoedijk** [comment, marked as answer]
Releases are manual. @mburumaxwell is the project maintainer, I just contribute here. He normally does a release every week or so when there are pending changes; There will probably be a new one sometime this week.

Regarding testing yourself, there are two options:
1. [Run the task locally using `npm run start:V2`](https://github.com/tinglesoftware/dependabot-azure-devops/blob/main/docs/extension.md#running-the-task-locally)
2. [Build a local copy of the extension and then publish and share it with your organisation](https://github.com/tinglesoftware/dependabot-azure-devops/blob/main/docs/extension.md#installing-the-extension)

Both of these require a bit of effort if you don't already have a development environment setup, it might be easier to wait for a new release.

**Reply by mburumaxwell** [reply]
You could also add your organization to the list that gets changes from the main branch.
https://github.com/tinglesoftware/dependabot-azure-devops/blob/b27101f53ee892aef84dc658f46ec45663ab130c/.github/workflows/extension.yml#L148-L155
